Constellation/Node/Api.hs

```diff
@@ -113,6 +139,38 @@ data Whitelist = Whitelist
     , wlIPv6 :: Set HostAddress6
     } deriving Show
 
+decodeSendRaw :: BL.ByteString -> RequestHeaders -> Either String Send
+decodeSendRaw b h = case getHeaders [hContentLength, hFrom, hTo] h of
+    Right headers -> Right Send
+                    { sreqPayload = decodePayload (hmap ! hContentLength) b
+                    , sreqFrom    = mustDecodeB64PublicKey $ hmap ! hFrom
```

Comment:
since they're only used here and in the unit test, should use just "from" and "to" here